precompile/allowlist/config.go

```diff
@@ -21,8 +21,8 @@ type AllowListConfig struct {
 }
 
 // Configure initializes the address space of [precompileAddr] by initializing the role of each of
-// the addresses in [AllowListAdmins].
-func (c *AllowListConfig) Configure(chainConfig precompileconfig.ChainConfig, precompileAddr common.Address, state contract.StateDB, blockContext contract.ConfigurationBlockContext) error {
+// the addresses in [AllowListConfig].
```

Comment:
Could we keep the arguments for this method for symmetry with the other methods (even though unused)?

Reply:
I'm not sure I understand what symmetry you are thinking about? With the `Configurator.Configure`??

Reply:
That was my thought, even though the AllowList is not a precompile it follows through the phases of Configure along with the precompile.

Reply:
Related question: what would be the cost of changing the signature of AllowListConfig.Configure if needed in the future? 🤔

Reply:
Less than the other method, as the AllowList is not part of the precompile interface but more a partial implementation that's used by some of the precompiles.

As I mentioned I have a subjective preference for methods that all take the same arguments but this is not a strong preference.